src/main/java/me/totalfreedom/totalfreedommod/bridge/EssentialsBridge.java

```diff
@@ -136,4 +152,131 @@ public boolean isEssentialsEnabled()
         }
         return false;
     }
+
+    public Boolean isVanished(String username)
```

Comment:
Quoting stackoverflow: "boolean is a java primitive type whereas Boolean is an object/reference type that wraps a boolean" To return null it has to be an Object, because boolean can only return true or false.